std/range/package.d

```diff
@@ -10923,6 +10923,33 @@ that break its sorted-ness, `SortedRange` will work erratically.
     a.sort.sort!"a > b";
 }
 
+/**
+Determines whether `T` is a $(LREF SortedRange).
+
+Params:
+    T = type to test
+
+Returns:
+`true` if `T` is a $(LREF SortedRange);
+`false` otherwise
+*/
+template isSortedRange(T) {
```

Comment:
Several issues with this:

* New public symbol added as a small part of an unrelated PR
* Wrong API - the predicate should be present (sorted by what?)
* Not actually needed in the PR so it needs to be justified separately (my take: unneeded)

Reply:
I think this addition is needed since there are other places that check whether a range is sorted ([1] is an example and there might be others), however, I adding it in this PR is not the right way to go. Regarding the presence of the predicate, I think that there are cases where you don't really care what the predicate is, but we should indeed accommodate the case where the predicate is also accounted for (in [1], it is also checked that the predicate is the default one).

What I suggest is to make a PR that adds `isSortedRange` so that others in the community take a look at it. Once it is merged we can come back to this one. How does that sound @burner ?

[1] https://github.com/dlang/phobos/blob/master/std/algorithm/searching.d#L1597